### ISSUE TITLE
Migrate ol-home to ol-home0

### ISSUE THREAD

**Issue by cdrini** [internetarchive/openlibrary#4060]
## Sub-tasks

- [x] #4058 Create ol-home0
- [x] #4059 Move infobase to ol-home0
- [x] Switch infogami submodule to infogami master https://github.com/internetarchive/openlibrary/issues/4249#issuecomment-744639377
- [x] #4287 Move solrupdater to ol-home0
    - [x] TODO: Deploy it #4395
- [x] #4324 Move ImportBot to ol-home0 --> `make-a-service` - Port ID?, Nagios,
- [x] internetarchive/infogami#134 Use empty str instead of None in HTTP headers
- [x] #4383 Move cron jobs to ol-home0 (Make one service for all cron jobs -- Caution on the double run)
   - [x] internetarchive/olsystem#110 Replace references of ol-home with ol-home0
   - [x] internetarchive/olsystem#118 DRAFT: Fix cron job oldump to run in Docker
   - [x] #4707 cron-jobs service stuck in restart loop
- [x] #4318 Deployments should be triggered from ol-home0
    - [x] #4395 Scripts for production deployments
    - [x] #4457 Reboot Docker services on all production servers

[Deployment Scratchpad](https://github.com/internetarchive/openlibrary/wiki/Deployment-Scratchpad)

**Comment by cclauss** [comment]
Two versions of solorupdater -- We want to put into separate docker service -- Create a new service in parallel to home service - Get starting command from home. 

Push local changes into that service.

**Comment by cclauss** [comment]
Questions:
* What is the status of / action plan for #2032 ?
* What is meant by [`deploys vs build directories`](https://github.com/internetarchive/openlibrary/issues/4249#issuecomment-744639377)
* Does [fabfile.rsync_repo()](https://github.com/internetarchive/olsystem/blob/master/fabfile.py#L211) work as is?

**Comment by cdrini** [comment]
Is it ok if we revert this back to be about ol-home? I think we were swapping the places of #4060 and #4318 . I've organized them as:

- This issue (#4060) -> Goal is deprecate ol-home. Lists all the services that will need to be moved over.
- #4318 Deployments should be triggered from ol-home0 -> goal is the command we run for deploying should happen on ol-home0, not ol-home
- #4382 Improve docker-based environment/deployment -> Our docker-based deploys are super manual and a bit of a mess :P Goal is to fix that.

**Comment by mekarpeles** [comment]
The remaining issues are as follows, which are being moved to a followup issue:
- [ ] Remove infobase ol-home:7000 -> ol-home0:7000 reverse proxy
    - [ ] remove ol-home proxypass to ol-home0 for infobase + update config
- [ ] Can we dump stats-solr and move from ol-home to ol-home0?
    - [ ] #4426 Can we remove it?
- [ ] can we reclaim 4gb of RAM from ol-home (until we can decommission)
- [ ] INCOMPLETE; anything else?
- [ ] Remove ol-home
